pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -1215,3 +1220,164 @@ def test_all(self):
             RPM_FIXTURE_SUMMARY,
             get_content_summary(repo.to_dict())
         )
+
+
+class DistributionTreeTest(PulpTestCase):
```

Comment:
Optional suggestion here but may want to break out this into a separate file since test_sync.py is getting incredibly long.

Reply:
I agree it is good idea, so moving test to test_distribution_tree.py